src/renderer/MMM-awesome-alexa.ts

```diff
@@ -34,7 +34,6 @@ Module.register("MMM-awesome-alexa", {
         this.sendSocketNotification("CONFIG", this.config);
         this.canvas = this.createCanvas();
         this.visualizer = new CircleVisualizer(this.canvas);
```

Comment:
Looks like that if this line is called when `isSpeechVisualizationEnabled` is `false` you can't hear Alexa's responses anymore.
So fix would be to wrap it to
```
if (this.config.isSpeechVisualizationEnabled) {
  this.visualizer = new CircleVisualizer(this.canvas);
}
```

Reply:
Oh sorry, didnt see this, fixing ASAP